includes/admin/class-wc-rest-stripe-settings-controller.php

```
@@ -689,9 +702,24 @@ public function dismiss_notice( WP_REST_Request $request ) {
 	}
 
+	/**
+	 * Moves Stripe gateways to the first positions in WooCommerce gateway order.
+	 *
+	 * @return WP_REST_Response
+	 */
+	public function set_stripe_gateways_first() {
+		WC_Stripe_Helper::move_stripe_gateways_to_top_in_woocommerce_gateway_order();
+
+		return new WP_REST_Response( [ 'result' => 'Stripe moved to first position' ], 200 );
+	}
```

PHPMD (2.15.0)

[error] The method set_stripe_gateways_first is not named in camelCase. (undefined)

(CamelCaseMethodName)